lib/android_apk.rb

```diff
@@ -83,25 +83,20 @@ module Reason
 
   extend Forwardable
 
-  delegate %i(label default_icon_path test_only test_only? package_name version_code version_name min_sdk_version target_sdk_version icons labels) => :@aapt2_result
-
-  alias icon default_icon_path
+  attr_reader :aapt2_badging_result, :icon_path_hash, :app_signature
```

Comment:
Aggregated `attr_reader` declaration and rename `aapt2_result` to `aapt2_badging_result`